From ad2565c8ff9c4309f2e5aacbd24856b615f20478 Mon Sep 17 00:00:00 2001
From: Kyle McMartin <kyle@redhat.com>
Date: Tue, 23 Jul 2013 22:07:05 -0400
Subject: [PATCH] dracut: 90kernel-modules/module-setup.sh: add ehci-tegra.ko
 on arm

kernel-tegra is now part of the base kernel package, so bits and pieces
ended up modular, and as a result, if you boot off the internal USB, you
drop to a dracut shell with no way of getting to root.
---
 modules.d/90kernel-modules/module-setup.sh | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/modules.d/90kernel-modules/module-setup.sh b/modules.d/90kernel-modules/module-setup.sh
index 8388a08..3d4048a 100755
--- a/modules.d/90kernel-modules/module-setup.sh
+++ b/modules.d/90kernel-modules/module-setup.sh
@@ -47,7 +47,8 @@ installkernel() {
         if [[ "$(uname -p)" == arm* ]]; then
             # arm specific modules
             hostonly='' instmods sdhci_esdhc_imx mmci sdhci_tegra mvsdio omap omapdrm \
-                omap_hsmmc panel-tfp410 sdhci_dove ahci_platform pata_imx sata_mv
+                omap_hsmmc panel-tfp410 sdhci_dove ahci_platform pata_imx sata_mv \
+                ehci-tegra
         fi
 
         # install virtual machine support
-- 
1.8.3.1

